packages/manager/cypress/support/intercepts/firewalls.ts

```diff
@@ -12,3 +12,16 @@ import { apiMatcher } from 'support/util/intercepts';
 export const interceptCreateFirewall = (): Cypress.Chainable<null> => {
   return cy.intercept('POST', apiMatcher('firewalls'));
 };
+
+/**
+ * Intercepts PUT request to update a Firewall.
+ *
+ * @param firewallId the ID of the Firewall to update
+ * @returns Cypress chainable.
+ */
+export const interceptUpdateFirewall = (
+  firewallId: string
+): Cypress.Chainable<null> => {
+  // return cy.intercept('PUT', apiMatcher('firewalls'));
+  return cy.intercept('PUT', apiMatcher(`firewalls/${firewallId}/rules`));
```

Comment:
```suggestion
  return cy.intercept('POST', apiMatcher('networking/firewalls'));
};

/**
 * Intercepts PUT request to update a Firewall.
 *
 * @param firewallId the ID of the Firewall to update
 * @returns Cypress chainable.
 */
export const interceptUpdateFirewall = (
  firewallId: string
): Cypress.Chainable<null> => {
  // return cy.intercept('PUT', apiMatcher('firewalls'));
  return cy.intercept('PUT', apiMatcher(`networking/firewalls/${firewallId}/rules`));
```
Sorry for not catching this sooner! I think all of our [Firewall API endpoints](https://www.linode.com/docs/api/networking/) are prefixed with `networking/` so we'll probably want that included here.